README.md

```diff
@@ -77,104 +53,95 @@ Table of Contents
+couple of cycles simulation time.)
+
+
+### Build Model and Run Simulations
+
+Build the Verilator model of CVA6 by using the Makefile:
 ```
-$ make verilate
+make verilate
 ```
 
 To build the verilator model with support for vcd files run
 ```
-$ make verilate DEBUG=1
+make verilate DEBUG=1
 ```
 
-This will create a C++ model of the core including a SystemVerilog wrapper and link it against a C++ testbench (in the `tb` subfolder).
-The binary can be found in the `work-ver` and accepts a RISC-V ELF binary as an argument, e.g.:
+This will create a C++ model of the core including a SystemVerilog wrapper and link it against a C++ testbench (in the `tb` subfolder). The binary can be found in the `work-ver` and accepts a RISC-V ELF binary as an argument, e.g.:
 
 ```
-$ work-ver/Variane_testharness rv64um-v-divuw
+work-ver/Variane_testharness rv64um-v-divuw
 ```
 
-The Verilator testbench makes use of the `riscv-fesvr`.
-This means that you can use the `riscv-tests` repository as well as `riscv-pk` out-of-the-box.
-As a general rule of thumb the Verilator model will behave like Spike (exception for being orders of magnitudes slower).
+The Verilator testbench makes use of the `riscv-fesvr`. This means that you can use the `riscv-tests` repository as well as `riscv-pk` out-of-the-box. As a general rule of thumb the Verilator model will behave like Spike (exception for being orders of magnitudes slower).
 
-Both the Verilator model as well as the Questa simulation will produce trace logs.
-The Verilator trace is more basic but you can feed the log to `spike-dasm` to resolve instructions to mnemonics.
-Unfortunately value inspection is currently not possible for the Verilator trace file.
+Both, the Verilator model as well as the Questa simulation will produce trace logs. The Verilator trace is more basic but you can feed the log to `spike-dasm` to resolve instructions to mnemonics. Unfortunately value inspection is currently not possible for the Verilator trace file.
 
 ```
-$ spike-dasm < trace_hart_00.dasm > logfile.txt
+spike-dasm < trace_hart_00.dasm > logfile.txt
 ```
 
-To build, compile and run the CVA6 core-only in its example testbench using Verilator (known to work with V4.108):
-```
-$ cd core/example_tb
-$ make veri_run
-```
-`make help` will print all supported targets.
-
-
 ### Running User-Space Applications
 
-It is possible to run user-space binaries on CVA6 with `riscv-pk` ([link](https://github.com/riscv/riscv-pk)).
+It is possible to run user-space binaries on CVA6 with ([RISC-V Proxy Kernel and Boot Loader](https://github.com/riscv/riscv-pk)). 
+RISC-V PK can be installed by running: `./ci/install-riscvpk.sh`
 
 ```
-$ mkdir build
-$ cd build
-$ ../configure --prefix=$RISCV --host=riscv64-unknown-elf
-$ make
-$ make install
+mkdir build
+cd build
+../configure --prefix=$RISCV --host=riscv64-unknown-elf
+make
+make install
 ```
 
 Then to run a RISC-V ELF using the Verilator model do:
 
 ```
-$ echo '
+echo '
```

Comment:
I don't see why you would remove the shell prompt everywhere. Revert imo.

Reply:
nevermind I guess copy pasting is a valid use case

Reply:
Yes, especially since it is natively offered by github i.e. the copy function, see on the right hand side
![image](https://user-images.githubusercontent.com/20418060/135533444-c319238f-943b-4cbb-a1ce-c72c6a2c687e.png)